### PR TITLE
bundle update deps

### PR DESCRIPTION
**Pull request by igorwwwwwwwwwwwwwwwwwwww**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -7,12 +7,12 @@ gem 'addressable', '~> 2.3'
 gem 'coder'
 gem 'jemalloc'
 gem 'metriks', '0.9.9.6'
-gem 'metriks-librato_metrics', github: 'eric/metriks-librato_metrics'
+gem 'metriks-librato_metrics', git: 'https://github.com/eric/metriks-librato_metrics'
```
Comment (GrahamCampbell) [inline Gemfile]: Why does this need to change?

Comment (igorwwwwwwwwwwwwwwwwwwww) [inline Gemfile]: The reason I made this change is that https provides transport security, whereas the git protocol does not.

Comment (GrahamCampbell) [inline Gemfile]: The git protocol does use transport security?

Comment (GrahamCampbell) [inline Gemfile]: https://git-scm.com/book/ch4-1.html#The-Git-Protocol

Comment (igorwwwwwwwwwwwwwwwwwwww) [inline Gemfile]: No, the git protocol is unauthenticated. It does not use SSH, and it does not use TLS. Here is a wireshark capture showing the plaintext information on the wire, from running `git clone git://github.com/travis-ci/travis-support.git`:

screen shot 2016-06-30 at 18 18 02

Comment (GrahamCampbell) [inline Gemfile]: What about `git clone git@github.com:travis-ci/travis-support`?

Comment (igorwwwwwwwwwwwwwwwwwwww) [inline Gemfile]: That's a clone over the SSH protocol -- which would also work as an alternative to `https`. However, that requires SSH to be setup and associated with a GitHub account.

**Pull request merged**